cli-wrap/src/main/resources/cli/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+
+pversion=`python3 -V`
+echo $pversion
+if [ "$pversion" != "Python 3.5.3" ] ; then
+    echo "Installing python 3"
+fi
+#cd ~
+#sudo yum install openssl-devel bzip2-devel expat-devel gdbm-devel readline-devel sqlite-devel
+#wget https://www.python.org/ftp/python/3.5.2/Python-3.5.2.tgz
+
+#tar -zxvf Python-3.5.2.tgz 
+#cd Python-3.5.2
+
+#./configure --prefix=/opt/python3 
+#make 
+#sudo make install
+
+#sudo dnf install python3-requests
+#sudo dnf install python3-websockets
+
```

Comment:
python3 may be already present, i would find if there exists python3.5 command, and then download and use ./configure without the prefix which will install python3.5
Also use pip instead of dnf
sudo pip3.5 install requests
sudo pip3.5 install websockets